pkg/beholder/config.go

```diff
@@ -17,8 +18,9 @@ type Config struct {
 	// Message Emitter
 	EmitterExportTimeout time.Duration
 	// OTel Trace
-	TraceSampleRate   float64
-	TraceBatchTimeout time.Duration
+	TraceSampleRate    float64
+	TraceBatchTimeout  time.Duration
+	TracerProviderOpts []sdktrace.TracerProviderOption
```

Comment:
Supporting configured `TracerProviderOption`s let's us pass in our legacy trace exporter.